trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name[0...10]
```

Comment:
- [ ] According to the project requiremnet you should trim to a maximum of 10, currently, you are timing to a maximum of 11 characters, so kindly change this to  ` @nameable.correct_name[0...9]`

Reply:
Hi @VICTOR-CHUKWU,

Thanks for the feedback. I have updated the code accordingly.

Best Regards,
K. Inoulen Shanongba